pepper-apis/src/main/java/org/broadinstitute/ddp/route/UserRegistrationRoute.java

```diff
@@ -382,7 +382,7 @@ private User handleGovernancePolicy(Response response, UserRegistrationPayload p
             shouldCreateGoverned = policy.shouldCreateGovernedUser(handle, interpreter, operatorUser.getGuid());
         } catch (Exception e) {
             String msg = "Error while evaluating study governance policy for study " + policy.getStudyGuid();
-            LOG.warn(msg, e);
+            LOG.error(msg, e);
```

Comment:
These are the only two places in this route where we're returning a server 500 response. In these two cases, we should log an error so we can get alerted about it.